bin/cmd.js

```diff
@@ -110,6 +112,31 @@ gcr.load(parsed, function(err) {
 
   if (!gcr.config.get('token')) {
     questions.push(tokenQuestion)
+    if (!gcr.config.get('registerToken')) {
+	questions.push(tokenQuestion)
+    } else {
+	    var fp = gcr.config.get('keypath')
```

Comment:
please use 2 spaces instead of tabs to match up with the rest of the repository.  I'll have to see about getting this added to the linting rules, but i don't know about lilint.  @evanlucas?